src/pages/Fox/components/FoxWifHatHoldingLine.tsx

```diff
@@ -0,0 +1,87 @@
+import type { StackDirection } from '@chakra-ui/react'
+import { Button, Flex, HStack, Stack, Text, useColorModeValue } from '@chakra-ui/react'
+import { useTranslate } from 'react-polyglot'
+import { WalletIcon } from 'components/Icons/WalletIcon'
+import { middleEllipsis } from 'lib/utils'
+
+type FoxWifHatHoldingLineProps = {
+  accountId: string
+  accountNumber: number
+  amount: string
+  symbol: string
```

Comment:
preferably-blocking: prefer passing `assetId`/ `asset` down and select it, which can then also be used to retrieve precision in case of passing a base unit amount down

Reply:
Done in [a79f134](https://github.com/shapeshift/web/pull/8605/commits/a79f1341343395bca74b469a901a3a4e12a71be0), used fox assetId waiting for foxwifhat deployment